cloudscheduler/openstackcluster.py

```diff
@@ -293,8 +334,8 @@ def vm_destroy(self, vm, return_resources=True, reason=""):
         except novaclient.exceptions.NotFound as e:
             log.error("VM %s not found on %s: removing from CS" % (vm.id, self.name))
         except Exception as e:
-            try:
-                log.error("Unhandled exception while destroying VM on %s : %s" % (self.name,e))
+                try:
```

Comment:
These two lines got accidentally additional spaces it seems

Reply:
how did those get there, hmmm